Classes/ViewHelpers/Widget/Controller/AzController.php

```diff
@@ -72,48 +72,45 @@ public function initializeAction()
      */
     public function indexAction()
     {
-
+        $excludeTheseCategories = [];
+        if (isset($this->settings['excludeTheseCategories']) && !empty($this->settings['excludeTheseCategories'])) {
+            $excludeTheseCategories = explode(',' , $this->settings['excludeTheseCategories']);
+        }
         // merge configuration with custom adoptions
         $customConfiguration = [
-            'titles' => $this->getAzGrouping($this->objects),
+            'titles' => $this->getAzGrouping($this->objects, $excludeTheseCategories),
             'linkPartialName' => $this->configuration['linkController'] . 'List',
             'cId' => $this->cObj->data['uid']
         ];
         $configuration = array_merge($customConfiguration, $this->configuration);
-
         $this->view->assignMultiple($configuration);
     }
 
     /**
      * Groups the titles to their indexes
      *
      * @param $titles
+     * @param array $excludeTheseCategories
      * @return array
      */
-    protected function getAzGrouping($titles)
+    protected function getAzGrouping($titles, $excludeTheseCategories)
```

Comment:
Set $excludeTheseCategories to a default value to not break calls from different extensions ...